src/release/breaking-changes/ignoringsemantics-migration.md

```diff
@@ -65,6 +92,46 @@ class _RenderIgnorePointerWithSemantics extends RenderProxyBox {
   @override
   bool hitTest(BoxHitTestResult result, { required Offset position }) => false;
 }
+
+/// A widget absorbs pointer event but still keeps semantics actions.
```

Comment:
```suggestion
/// A widget absorbs pointer events but still keeps semantics actions.
```
What do you mean by "keeps" semantic actions?

Reply:
updated the phrasing